frontend/packages/console-dynamic-plugin-sdk/src/utils/flags.ts

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+// eslint-disable-next-line @typescript-eslint/ban-ts-ignore
+// @ts-ignore
+import { useSelector, useDispatch } from 'react-redux';
+import { action } from 'typesafe-actions';
+import { RootState } from '@console/internal/redux';
```

Comment:
Better would be to use [SDKStoreState](https://github.com/openshift/console/blob/master/frontend/packages/console-dynamic-plugin-sdk/src/app/redux-types.ts#L22) type (and we can also get rid of `@console/internal` import)

Reply:
FLAGS is not part of it.